app/subscriber/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import React from 'react';
+import { FaAnglesLeft, FaAnglesRight } from 'react-icons/fa6';
+import { useNavigate } from 'react-router-dom';
+import { Row, Show } from 'tno-core';
+
+import { INavbarOptions, navbarOptions } from './NavbarItems';
+import * as styled from './styled';
+import { determineGroupIcon } from './utils/determineGroupIcon';
+
+export interface INavbarGroup {
+  groupName: string;
+  sortOrder: number;
+}
+
+export interface INavbarProps {
+  options?: INavbarOptions[];
+}
+
+export const Navbar: React.FC<INavbarProps> = ({ options }) => {
+  const [expanded, setExpanded] = React.useState(true);
```

Comment:
Keep your `use*` dependencies first.
Then `useState`
Then variables
Then `useEffect`

It makes it easier to read and consistent with all components

Reply:
Darn it tried to sneak that in before you saw, updated now lol